src/main/java/FileHandler.java

```diff
@@ -0,0 +1,48 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileHandler {
+
+    public static boolean isMarked(String input) {
+        return String.valueOf(input.charAt(4)).equals("X");
+    }
+    public static Path addTasks(ArrayList<Task> storage, Path filePath) throws IOException {
```

Comment:
Should the method be named addTasks? because it might be confusing as it has a similar method in TaskHandler